server/models/bhima.sql

```diff
@@ -98,7 +98,8 @@ INSERT INTO unit VALUES
   (204, 'Exchange Rate','TREE.EXCHANGE','',1,'/modules/exchange/exchange','/exchange'),
   (205, 'Account Reference Management','TREE.ACCOUNT_REFERENCE_MANAGEMENT','',1,'/modules/account_reference','/account_reference'),
   (206, '[OHADA] Bilan','TREE.OHADA_BALANCE_SHEET','',144,'/modules/reports/ohada_balance_sheet_report','/reports/ohada_balance_sheet_report'),
-  (207, 'Account Reference Report','TREE.ACCOUNT_REFERENCE_REPORT','',144,'/modules/reports/account_reference','/reports/account_reference');
+  (207, 'Account Reference Report','TREE.ACCOUNT_REFERENCE_REPORT','',144,'/modules/reports/account_reference','/reports/account_reference'),
```

Comment:
@jniles How should changes to core BHIMA tables be tracked for upgrading production servers? 

Is `bhima.sql` run whenever and upgrade is performed? It does include things that may have changed like `transaction_type`. Otherwise is there a method of tracking changes that don't require a schema update?

Reply:
Ugh.  Right now we just manually upgrade things.  I'm not sure of a better way, or how to make sure people respect the better way when writing new code.  As a reviewer, I find it hard to remember to remind people to make upgrade scripts.